src/mixins/filing-template-mixin.ts

```diff
@@ -375,6 +378,9 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
       dissolutionStatementType: draftFiling.dissolution?.dissolutionStatementType
     })
 
+    // // Take the first custodian, as there is only a singular custodian in a dissolution filing.
+    if (draftFiling.dissolution.parties) this.setCustodianOfRecords({ ...draftFiling.dissolution.parties[0] })
```

Comment:
This looks complicated. Would this work?
```
if (draftFiling.dissolution.parties) this.setCustodianOfRecords(draftFiling.dissolution.parties[0])
```

Reply:
oo yea, i think that simplifies it a bit. Will try and update.

Reply:
The theoretical advantage of destructuring and then objectifying is that the properties are copied (instead of the object being assigned) but I think the latter is OK.